GuildedRoseLLC/Items/ItemsViewController.swift

```diff
@@ -46,4 +46,16 @@ public class ItemsViewController: UIViewController, UICollectionViewDelegate {
         itemCollectionView.delegate = self
         
     }
+    
+    public override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
+        if segue.destination is DetailViewController && sender is UIButton {
```

Comment:
Given that there is an `if` statement here, I would expect more than one test for this function.

Reply:
added